### PR TITLE
new-account wallets in simplified UI

### PR DESCRIPTION
**Pull request by howard-stearns**
https://highfidelity.atlassian.net/browse/BUGZ-881

### CODE REVIEW

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml [hunk 1]
```diff
@@ -76,10 +76,20 @@ Rectangle {
                 return;
             }
 
-            if (walletStatus === 5) {
+            switch (walletStatus) {
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: for consistency with the rest of the SimplifiedUI `switch` statements, please indent all cases, as in:
```
            switch (walletStatus) {
                case 1:
                    var randomNumber = Math.floor(Math.random() * 34) + 1;
                    var securityImagePath = "images/" + randomNumber.toString().padStart(2, '0') + ".jpg";
                    Commerce.getWalletAuthenticatedStatus(); // before writing security image, ensures that salt/account password is set.
                    Commerce.chooseSecurityImage(securityImagePath);
                    Commerce.generateKeyPair()
                    Commerce.getWalletStatus();
                    break;
                case 5:
                    topBarInventoryModel.getFirstPage();
                    break;
                default:
                    console.log('WARNING: SimplifiedTopBar.qml walletStatusResult:', walletStatus);
                }
            }
```

**Comment by howard-stearns** [comment]
This is safe for merging, but I'd like to give Rick a chance to confirm that everything works, so that we don't thrash.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20291/HighFidelity-Beta-PR15904-4191c37.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20291/HighFidelity-Quest-Beta-PR15904-4191c37.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20291/HighFidelity-Beta-PR15904-4191c37.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20291/HighFidelity-Beta-Interface-PR15904-4191c37.dmg

**Comment by howard-stearns** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20295/HighFidelity-Beta-PR15904-4191c37.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20295/HighFidelity-Beta-Interface-PR15904-4191c37.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20295/HighFidelity-Beta-PR15904-4191c37.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20295/HighFidelity-Quest-Beta-PR15904-4191c37.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20295/HighFidelity-Beta-PR15904-4191c37.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20295/HighFidelity-Beta-Interface-PR15904-4191c37.exe

**Pull request merged**